target/linux/rtd1xxx/files-5.15/drivers/clk/realtek/Kconfig

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: GPL-2.0-only
+menuconfig COMMON_CLK_REALTEK
+	bool "Realtek Clock drivers"
+	depends on ARCH_REALTEK || COMPILE_TEST
+	default y
+
+if COMMON_CLK_REALTEK
+
+config RTK_CLK_COMMON
```

Comment:
we should figure out when to abbreviate,a nd when not. the two notitions in this line and the line above look a bit weird.

Reply:
COMMON_CLK_REALTEK is more like a global switch which controls other config options.

RTK_CLK_COMMON is bind to clk-rtk.o this module.

Probably we can remove RTK_CLK_COMMON and enable this clk-rtk.o by default in the Makefile ?